src/routes/RootRoute.module.css

```
@@ -0,0 +1,25 @@
+.locationOverlay {
```

You may use the standard kebab-case here, the class names will be translated to camelCase for use in TS
```suggestion
.location-overlay {
```